src/store/board.store.js

```diff
@@ -1,25 +1,71 @@
-import {writable, get, readable} from 'svelte/store'
+//@ts-check
+import {writable, get, derived, Writable} from 'svelte/store'
+/**
+ * @typedef {Object} CardModel
+ * 
+ * @typedef {Object} ColumnModel
+ * @property {number} id
+ * @property {string} title
+ * @property {Array<CardModel>} cards 
+ */
 
+/** @type {Writable<ColumnModel[]>} */
 export const board = writable([])
+export const lastId = derived(board, ($board, set) => {
```

Comment:
To be honest it shouldn't be exported. I think no component will need this store. It's only a utility